cloudkittyclient/v1/report/shell.py

```diff
@@ -123,10 +128,11 @@ def do_invoice_show(cc, args):
            required=False)
 
 def do_invoice_get(cc, args):
-    """Get invoice based on tenant-id, invoice-id and payment_status args."""
+    """Get invoice based on tenant-id, invoice-id, tenant-name and payment_status args."""
 
     # Invoice details as Unicode response
     invoice = cc.reports.get_invoice(tenant_id=args.total_tenant_id,
+				   tenant_name1=args.total_tenant_name,
```

Comment:
Why use tenant_name1 here and above?  Seems like this should be tenant_name as there is plan for multiple tenant names.